### ISSUE TITLE
Import { ... } as wildcard

### ISSUE THREAD

**Issue by tristan-shelton** [vihanb/babel-plugin-wildcard#1]
I'm using this plugin in my babel/webpack/React workflow and it's fantastic! One thing I would like to see is the ability to use the specific import syntax with directories because I don't want my imports inside an object.

Currently I do this:
```javascript
import * as things from 'src/things'
const { ThingA, ThingB, ThingC } = things
```

And that works well enough, but I would like to be able to do this:

```javascript
import { ThingA, ThingB, ThinbC } from 'src/things'
```

Thanks for writing this wonderful little plugin!

**Comment by vihanb** [comment]
I added this feature in 155d89bb101e523f2542b47ced3632c8059af90e and it is on npm as part of v2.0.0 (you may need to update). You have to explicitly specify it's a wildcard though: `import { ThingA, ThingB, ThingC } from 'src/things/*'` It looks like it's working in the tests but since this is a big change let me know if it doesn't work

**Comment by tristan-shelton** [comment]
I was indeed using the wrong version. Thanks! 👍

**Comment by tristan-shelton** [comment]
I finally got around to trying this out and it doesn't seem to work correctly in a simple test case.

Here's what I did:

* `mkdir wildcard-test && cd wildcard-test`
* `npm init`
* `npm i -S babel-cli babel-plugin-wildcard`
* `mkdir things`
* `touch things/thing{1..3}.js`
* `echo "import { Thing1, Thing2, Thing3 } from './things/*'" > test.js`
* `babel -o out.js test.js`

The result I get is a file containing `import { Thing2 } from './things';` which definitely doesn't seem right! 😄

**Comment by tristan-shelton** [comment]
Just one more quick note to confirm that the main function of the plugin is working as intended in the same test case:

```
$ cat test.js
import { Thing1, Thing2, Thing3 } from './things/*'
$ echo "import * as Things from './things'" > test.js
$ babel --plugins=wildcard -o out.js test.js
$ cat out.js
const Things = {};
import _wcImport3 from './things/thing3.js';
Things.Thing3 = _wcImport3;
import _wcImport2 from './things/thing2.js';
Things.Thing2 = _wcImport2;
import _wcImport from './things/thing1.js';
Things.Thing1 = _wcImport;
```

**Comment by vihanb** [comment]
This appears to have been a really hidden bug :P but I think I found it . I have this loop which goes over all the specifiers (Thing1, Thing2, Thing3):

```js
// Here, node.specifiers is [Thing1, Thing2, Thing3]
for (var i = 0; i < node.specifiers.length; i++) {
    // Here only Thing1 & Thing3 are ever logged
```

So it looks like I did a `node.specifiers.splice(i, 1)` to remove the specifier but since I am not iterating backwards, It'll skip over Thing2, when Thing1 is removed. I'll have a fix on npm in ~10 minutes

**Comment by vihanb** [comment]
Fixed as of ea4ae7b36a6c122e3979adeef1d0913c04258517 just pushed to npm with version v2.1.1

**Comment by tristan-shelton** [comment]
Seems to work! Thanks so much for the great work. This plugin saves me so much effort. 👍
